src/modules/python.rs

```diff
@@ -1,18 +1,24 @@
 use super::Segment;
 use crate::context::Context;
+use crate::find_file;
 use ansi_term::Color;
-use std::path::PathBuf;
 use std::process::Command;
 
 /// Creates a segment with the current Python version
 ///
 /// Will display the Python version if any of the following criteria are met:
 ///     - Current directory contains a `.py` file
-///     - Current directory contains a `.python-version` file
+///     - Current directory contains a `.pyproject.toml` file
 ///     - Current directory contains a `requirements.txt` file
 ///     - Current directory contains a `pyproject.toml` file
 pub fn segment(context: &Context) -> Option<Segment> {
-    let is_py_project = context.dir_files.iter().any(has_py_files);
+    let python_criteria = find_file::Criteria {
+        files: vec!["requirements.txt, pyproject.toml", ".pyproject.toml"],
+        extension: "py".to_string(),
+        folder: "".to_string(),
```

Comment:
Since we won't always need all three criteria, maybe this would be best done as a builder pattern?